gallery/src/components/NxTable/NxTableFilterExample.tsx

```diff
@@ -77,7 +77,7 @@ const NxTableFilterExample = () => {
             <NxTableCell>Name</NxTableCell>
             <NxTableCell>Country</NxTableCell>
           </NxTableRow>
-          <NxTableRow>
+          <NxTableRow className="nx-table-row--filter-header">
             <NxTableCell isFilterHeader>
```

Comment:
This prop needs to be removed (both from the example and the `NxTableCell` implementation) and a similar prop needs to be created and used on `NxTableRow`.

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/91dbdc071887b85edc9bb5d745a65b16def34b86
And https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/036018c1c0d61d7168da9f3b5ab3f800710705d9

Reply:
> Done: 91dbdc0
And 036018c

That's added it to `NxTableRow` but not removed the outdated code from `NxTableCell`

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/88a744b970516433c36daf645de8a075190f281a